### DISCUSSION TITLE
"Missing trie node" & "required historical state unavailable" in logs

### DISCUSSION THREAD

**Discussion by FelixLai0926** [blockscout/blockscout#10719] [Q&A]
I am getting the following logs in my blockscout log

`0x0000000000000000000000000000000000000000@19632688: (-32000) missing trie node 1073cff865141fd8c38627b2f6a259a150f228478eec7953b4e8ebbb4ab9536a (path ) `

My geth start command looks like

`geth --networkid 1122 --http --http.port=8545 --http.addr=0.0.0.0 --http.vhosts=* --http.api "eth,net,web3,debug,txpool" --ws --ws.addr "0.0.0.0" --ws.port 8546 --ws.api "eth,net,web3,debug,txpool" --syncmode full --gcmode archive`

geth version is v1.10.25

**Comment by GimluCom** [comment]
Hi, has your geth node been run without `--syncmode full --gcmode archive` at some point? If so, you would likely need to set up a new node to sync from scratch in order to get a functional archive node.

**Reply by FelixLai0926** [reply]
But my node has always been running with `--syncmode full --gcmode archive`.

**Reply by ekawibisono** [reply]
> But my node has always been running with `--syncmode full --gcmode archive`.

hello, can you try use active envs blockscout TRACE_FIRST_BLOCK and FIRST_BLOCK ?

**Comment by chrischarlesharrison** [comment]
Also getting this issue

**Reply by GimluCom** [reply]
Would need some more details order help. Preferably reach out at [discord.gg/blockscout](https://discord.gg/blockscout)
